homeassistant/components/yeelight/light.py

```diff
@@ -85,21 +95,58 @@ def _wrap(self, *args, **kwargs):
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
+    from yeelight.enums import PowerMode
+
+    PLATFORM_DATA_KEY = '{}_lights'.format(DOMAIN)
```

Comment:
Make the variable lowercase. Constants are uppercase but should be defined at the module level. We don't do that here.

Reply:
Fixed